tests/integration_tests/test_depletereactor.py

```diff
@@ -121,9 +121,9 @@ def test_resources(self):
         tbl = self.resources
         times = self.to_array(tbl, "TimeCreated")
         quantities = self.to_array(tbl, "Quantity")
-        assert len(tbl) == 6
-        assert all(times == [0,0,0,2,5,8])
-        assert all(quantities == [10]*6)
+        assert len(tbl) == 9
+        assert all(times == [0,0,0,2,2,5,5,8,8])
+        assert all(quantities == [10]*9)
```

Comment:
```suggestion
        assert all(quantities == [10]*len(tbl))
```
Would it reduce the chance for errors if you add times later if you changed it to this? For test_resources too?

Reply:
This part of the code is testing what's in the output database. So your code suggestion is just testing if all of the entries in the table are of value 10, not specifically if there are 9 occurrences of value 10 in the table. I suppose that by testing for `[10]*9` that's also checking that there are 9 values in the table, same as l.124. but I'm okay with being redundant in my testing.